docs/tutorials/gromacs/spack-gromacs.md

```diff
@@ -0,0 +1,287 @@
+# HPC Toolkit - Install and Run Gromacs on a Slurm Cluster
+
+HPC Toolkit is an open-source software offered by Google Cloud which makes it
+easy for customers to deploy HPC environments on Google Cloud.
+
+In this tutorial you will use the HPC Toolkit to:
+
+* Deploy a [Slurm](https://github.com/SchedMD/slurm-gcp#readme) HPC cluster on
+  Google Cloud
+* Use [Spack](https://spack.io/) to install the Gromacs application and all of
+  its dependencies
+* Run a [Gromacs](https://www.gromacs.org/) job on your newly provisioned
+  cluster
+* Tear down the cluster
+
+## Select a Project
+
+Select a project in which to deploy an HPC cluster on Google.
+
+<walkthrough-project-setup billing="true"></walkthrough-project-setup>
+
+Once you have selected a project, click START.
+
+## Enable APIs & Permissions
+
+_Skip this step if you already ran this as part of a previous tutorial._
+
+In a new Google Cloud project there are several apis that must be enabled to
+deploy your HPC cluster. These will be caught when you perform `terraform apply`
+but you can save time by enabling them now by running:
+
+<!-- Tried the native way to do this and it timed out. Leaving comment here for future reference. -->
+<!-- <walkthrough-enable-apis apis="file.googleapis.com,compute.googleapis.com"></walkthrough-enable-apis> -->
+
+```bash
+gcloud services enable --project <walkthrough-project-id/> file.googleapis.com compute.googleapis.com
+```
+
+We also need to grant the default compute service account project edit access so
+the slurm controller can perform actions such as auto-scaling.
+
+<!-- Tried getting PROJECT_NUMBER using <walkthrough-project-number/> but returns empty string. -->
+
+```bash
+PROJECT_NUMBER=$(gcloud projects list --filter=<walkthrough-project-id/> --format='value(PROJECT_NUMBER)')
+
+echo "granting roles/editor to $PROJECT_NUMBER-compute@developer.gserviceaccount.com"
+
+gcloud iam service-accounts enable --project <walkthrough-project-id/> "$PROJECT_NUMBER"-compute@developer.gserviceaccount.com
+
+gcloud projects add-iam-policy-binding <walkthrough-project-id/> --member=serviceAccount:"$PROJECT_NUMBER"-compute@developer.gserviceaccount.com --role=roles/editor
+```
+
+## Build the Toolkit Binary
```

Comment:
The instructions are written in such a way to only use the locally built binary (aka never calls `make install`). So as long as they are following along they will be using a freshly built binary.

Reply:
Sorry, I more meant that the instructions say "Don't do this if you've done it in a previous lab" and I am not really sure what happens if you click "Open in cloud shell" multiple times, with different branches. Does it properly fetch the new remote, and then check out the branch, so `make` would build the right version of the binary for the lab you're trying to test? Or would it just leave the directory alone and expect you to manually update it?

Honestly, this is not really a big deal for this tutorial right now, but something we should figure out to make sure these remain robust longer term.

Reply:
Every time you click the buttons it clones a new copy of the repo, so you probably need to `make` each time. Even if it is the same repo, it does not hurt to call make. I am just going to delete the note that says "skip this"

In fact, all the calls are idempotent so I may remove the note on the previous section as well. I am worried it will cause more confusion that help.